src/elements/content-explorer/ItemGrid.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import React from 'react';
+import getProp from 'lodash/get';
+import AutoSizer from 'react-virtualized/dist/es/AutoSizer';
+import GridViewWrapper from '../../components/grid-view/GridViewWrapper';
+import ItemGridCell from './ItemGridCell';
+
+type Props = {
+    canDelete: boolean,
+    canDownload: boolean,
+    canPreview: boolean,
+    canRename: boolean,
+    canShare: boolean,
+    currentCollection: Collection,
+    isSmall: boolean,
+    isTouch: boolean,
+    onItemClick: Function,
+    onItemDelete: Function,
+    onItemDownload: Function,
+    onItemPreview: Function,
+    onItemRename: Function,
+    onItemSelect: Function,
+    onItemShare: Function,
+    rootElement?: HTMLElement,
+    rootId: string,
+    view: View,
+};
+
+const ItemGrid = ({
+    canDelete,
```

Comment:
You should be able to make use of `...rest` here, since the props are all defined in the Props type.